src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  background-color: #f6f6f6;
+}
+
```

Comment:
- [ ] I think it would be better if you add the checkbox as the design given as one of the requirements. Please fix this issue.  

😄 💪